src/igvfd/schemas/treatment.json

```diff
@@ -0,0 +1,206 @@
+        "duration"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/notes"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        }
+    ],
+    "dependentSchemas": {
+        "amount": {
+            "required": [
+                "amount_units"
+            ],
+            "comment": "Amount units must be specified if amount is specified"
+        },
+        "duration": {
+            "required": [
+                "duration_units"
+            ],
+            "comment": "Duration units must be specified if duration is specified"
+        },
+        "treatment_term_id": {
+            "oneOf": [
+                {
+                    "properties": {
+                        "treatment_type": {
+                            "enum": [
+                                "protein"
+                            ]
+                        },
+                        "treatment_term_id": {
+                            "pattern": "^((UniProtKB:[A-Z0-9]{6})|(NTR:[0-9]{2,8}))$"
+                        }
+                    }
+                },
+                {
+                    "properties": {
+                        "treatment_type": {
+                            "enum": [
+                                "chemical"
+                            ]
+                        },
+                        "treatment_term_id": {
+                            "pattern": "^((CHEBI:[0-9]{1,7})|(NTR:[0-9]{2,8}))$"
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "documents": {
+            "description": "Documents that describe the treatment protocol."
+        },
+        "amount": {
+            "title": "Amount",
+            "type": "number"
+        },
+        "amount_units": {
+            "title": "Amount units",
+            "type": "string",
+            "enum": [
+                "mg/kg",
+                "mg/mL",
+                "mM",
+                "ng/mL",
+                "nM",
+                "percent",
+                "μg/kg",
+                "μg/kg",
+                "μg/mL",
+                "μM"
+            ]
+        },
+        "duration": {
+            "title": "Time duration",
+            "type": "number"
+        },
+        "duration_units": {
+            "title": "Time duration units",
```

Comment:
please add description and comment explaining that duration needs to be specified